using-timescaledb/continuous-aggregates.md

```diff
@@ -181,9 +181,7 @@ SELECT drop_chunks(INTERVAL '30 days', 'device_readings')
 You can read more about data retention with continuous aggregates in
 the [*Data retention*][sec-data-retention] section.
 
->:TIP: Most times the continuous aggregate view will be updated by the background job;
-  however, if you would like to run it yourself, you may use the
-  [`REFRESH MATERIALIZED VIEW` command][api-refresh-continuous-aggs].
+>:TIP: Most times the continuous aggregate view will be updated by the background job; however, if you would like verify that the view is working correctly, you may use the [`REFRESH MATERIALIZED VIEW` command][api-refresh-continuous-aggs].
```

Comment:
The rest of PR uses multiple lines if sentences or paragraphs are too long, which is nice. Can it be fixed to into multilines here too, so it will be as originally and possible to track changes?

Reply:
I removed this paragraph for the time being since we consider `REFRESH MATERIALIZED VIEW` a debug command, but not sure what alternative we have for testing the view right now, so it might re-surface.